Kernel/Config/Files/XML/Ticket.xml

```diff
@@ -4256,6 +4256,13 @@
             <Item ValueType="String" ValueRegex="">%%Pending</Item>
         </Value>
     </Setting>
+    <Setting Name="Ticket::Frontend::AgentTicketActionCommon###RememberLastPendingTime" Required="0" Valid="1">
+        <Description Translatable="1">Defines if in input masks based on AgentTicketActionCommon a potentially set pending time should be prefilled if it is later than now.</Description>
```

Comment:
Restore last pending time in input masks based on AgentTicketActionCommon, if pending state is set.